packages/react-ui/src/atoms/Typography/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography as MuiTypography } from '@mui/material';
+
+const Typography = ({ italic = false, weight, children, ...otherProps }) => {
```

Comment:
I know it's typography PR, i expect we would have extended support for our `color`s, we've just got much more colors that MUI native `primary`, `secondary`, `textPrimary` ...

Reply:
Yes, we have indeed, you're gonna be able to do that, like this:

`  <Typography color='default.background'>
    {someText}
  </Typography>`